src/documents/FunctionGrades.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Function Grades
+layout: default
+category: Infusion
+---
+
+Most [grades](ComponentGrades.md) represent Infusion components - these are derived from the base grade `fluid.component`. However, some grades describe plain JavaScript functions - these are derived from the base grade `fluid.function`.
+The purpose of the Fluid API call [fluid.defaults](CoreAPI.md#fluid-defaults-gradename-options-) could be understood as providing *metadata* about some element of the system. In the case of a full component grade, this metadata is sufficient
+to allow the framework to automatically construct the creator function for the component. In the case of a function grade which describes an already existing function with a global name, the metadata supplies hints to the user about
+how to call the function and its purpose.
+
+## Registering a global function
+
+A global function is registered within infusion at a stable place in its global namespace by working with the core API functions [fluid.registerNamespace](CoreAPI.md#fluid-registernamespace-path-) and 
+[fluid.setGlobalValue](CoreAPI.md#fluid-setglobalvalue-path-value-) - in practice the latter is rarely used, in favour of directly setting members on namespaces objects.
```

Comment:
Should the last two words here be "namespaced object" (i.e. instead of "namespaces")?